.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 work
-*.iml
\ No newline at end of file
+*.iml
+.idea
```

Comment:
You can also define a global gitignore on your machine to avoid being bothered by IDE on repositories missing the full ignore list.